src/audio/SDL_audio.c

```diff
@@ -1357,6 +1357,9 @@ int SDL_BindAudioStreams(SDL_AudioDeviceID devid, SDL_AudioStream **streams, int
         return SDL_SetError("Audio streams are bound to device ids from SDL_OpenAudioDevice, not raw physical devices");
     } else if ((logdev = ObtainLogicalAudioDevice(devid)) == NULL) {
         return -1;  // ObtainLogicalAudioDevice set the error message.
+    } else if (logdev->is_simplified) {
```

Comment:
suggest `simplified` instead of `is_simplified`?

Reply:
Yes, but there's a bunch of these in SDL_sysaudio.h, so I'll clean that out separately.

Reply:
Resolved in 3699b12ed0a25b0667b23ed18fa0a60c800ede9d